<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Screen_12</class>
 <widget class="QWidget" name="Screen_12">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="d2_12_backgroundWidget" native="true">
     <property name="styleSheet">
      <string notr="true">#d2_12_backgroundWidget{
	background-color: rgb(255, 255, 255);
	border-radius:  22px;

	border-color: rgb(0, 0, 0);
	border-style: outset;
	border-width: 8px;
}

#d2_12_btnClose{
	border-image: url(:/desing2/buttons/close buttons/big/close-button-normal-big.png);
}

#d2_12_btnClose::pressed{
	border-image: url(:/desing2/buttons/close buttons/big/close-button-pressed-big.png);
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="d2_12_btnClose">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="d2_12_lblInfoText">
        <property name="font">
         <font>
          <family>Baloo Cyrillic</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Возврат на карту</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="d2_12_lblText_1">
        <property name="font">
         <font>
          <family>PT Sans Bold</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Внимание, возврат на карту обычно
 происходит мгновенно,
 но время может быть увеличено банком.</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="d2_12_lblText_2">
        <property name="font">
         <font>
          <family>PT Sans</family>
         </font>
        </property>
        <property name="text">
         <string>Для возврата средств приложите карту,
 с которой была оплата. </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
